src/app/payment/paymentMethod/PaymentMethodTitle.tsx

```diff
@@ -123,7 +123,7 @@ function getPaymentMethodTitle(
             },
             [PaymentMethodId.Mollie]: {
                 logoUrl: method.method === 'credit_card' ? '' : cdnPath(`/img/payment-providers/${method.method}.svg`),
-                titleText: startCase(methodName),
+                titleText: method.method === 'credit_card' ? startCase(methodName) : methodName,
```

Comment:
if the method is not `credit_card`, is it actually wrong to also upper case the first letter?

Reply:
Yes, for example `iDeal` should appear with the `i` as lowercase instead of `Ideal` or `IDeal`